### PR TITLE
update: Rename template generation flag

### PR DESCRIPTION
**Pull request by mrshu**
* Previously, the template generation flag was called `temp`. That did
  not say much about its fuction and so this commit changes that to
  `generate-from-template`.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func main() {
 		"detect a level from a given hash and home directory")
 	encrypt_flag := flag.Bool("enc", false, "encrypt a given challenge")
 	decrypt_flag := flag.Bool("dec", false, "decrypt a given challenge")
-	template_flag := flag.Bool("temp", false,
+	template_flag := flag.Bool("generate-from-template", false,
```
Comment (vidriduch) [inline main.go:25]: @mrshu Could we perhaps add a short version of this argument something like `-gen` or just `-g` so there would be a long version that is self explanatory and short version that is just easier to write when used from command line?

Comment (mrshu) [reply main.go:25]: @vidriduch Makes sense, `-g` should be easy to add

**Comment by mrshu** [comment]
@vidriduch I added the `-g` flag, can you please take another look?

**Comment by vidriduch** [comment]
@mrshu Ok, this look fine.

**Pull request merged**